package.json

```diff
@@ -1,15 +1,11 @@
 {
-  "name": "visual",
-  "scripts": {
-    "postinstall": "pbiviz update 1.6.0",
-    "pbiviz": "pbiviz",
-    "package": "pbiviz package",
-    "lint": "tslint -r \"node_modules/tslint-microsoft-contrib\" \"+(src)/**/*.ts\"",
-    "start": "pbiviz start"
-  },  
-  "devDependencies": {
-    "tslint": "^4.4.2",
-    "tslint-microsoft-contrib": "^4.0.0",
-    "powerbi-visuals-tools": "1.6.2"
-  }
-}
```

Comment:
Could you please restore this file? I suppose that we just need to update version of powerbi-visuals-tools here.